### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by jasonmadigan**
Some README updates to move us away from the standard template README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,25 +86,19 @@ to run in-cluster.
...
+### Via the `kuadrant-operator`
 
-```shell
-helm upgrade -i  my-plugin charts/openshift-console-plugin -n plugin__console-plugin-template --create-namespace --set plugin.image=my-plugin-image-location
-```
-
-NOTE: When deploying on OpenShift 4.10, it is recommended to add the parameter `--set plugin.securityContext.enabled=false` which will omit configurations related to Pod Security.
-
-NOTE: When defining i18n namespace, adhere `plugin__<name-of-the-plugin>` format. The name of the plugin should be extracted from the `consolePlugin` declaration within the [package.json](package.json) file.
+Install via the `kuadrant-operator`
```
Comment (david-martin) [inline README.md:101]: missing some steps here, or a link to kuadrant docs site?
Will the plugin be included out of the box with the kuadrnat operator?

File: README.md [hunk 2]
```diff
@@ -143,25 +86,19 @@ to run in-cluster.
...
+`oc apply -f install.yaml`
 
-Additional parameters can be specified if desired. Consult the chart [values](charts/openshift-console-plugin/values.yaml) file for the full set of supported parameters.
+or
 
-### Installing the Helm Chart
+`kubectl apply -f install.yaml`
 
-Install the chart using the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `plugin_console-plugin-template` parameter and providing the location of the image within the `plugin.image` parameter by using the following command:
+### Via the `kuadrant-operator`
```
Comment (jasonmadigan) [inline README.md:99]:
```suggestion
### Via the [`kuadrant-operator`](https://www.github.com/kuadrant/kuadrant-operator)
```

File: README.md (hunk 1 above)
Comment (jasonmadigan) [reply README.md]: added more words - will be included ootb with the operator

**Pull request merged**